### DISCUSSION TITLE
Allow changing popover offset

### DISCUSSION THREAD

**Discussion by islami00** [mantinedev/mantine#3594] [Feature requests]
This is related to the Popover component, 
I would like to use it to create a custom select dropdown where the dropdown looks like a. "section" of the input, but the popover does not allow for changing offset in the `x` or `y` direction.

I would like to use that to bring the popover to sit just in a place with the target element.

Attached is an example of what the desired behaviour is. Thanks!
<img width="305" alt="image" src="https://user-images.githubusercontent.com/76726548/220661381-059c79f7-f817-42ec-b7c3-b7dbb1aec3f9.png">

**Comment by islami00** [comment]
After doing some more digging, the menu component seems to have this option  https://mantine.dev/core/menu/#dropdown-position, and that fulfills my use case.

However, it is opinionated about accessibility (it has to be a menu) so this could still be a valid use case assuming someone wants to create a generic popover that isn't a menu, so I'm leaving this open with that in mind. Thanks for creating a great library!

**Reply by rtivital** [reply]
Menu component is built with Popover, it inherits all positioning props from Popover. Everything that is supported by menu is supported by Popover.

**Reply by islami00** [reply]
Wow, I completely missed that 😅 , thanks!
